io/net/iperf_test.py

```diff
@@ -80,6 +80,17 @@ def setUp(self):
             if not output.exit_status == 0:
                 self.cancel("unable to install the package %s on peer machine "
                             % pkg)
+        cmd1 = "systemctl stop firewalld.service"
+        cmd2 = "systemctl status firewalld.service"
```

Comment:
The same command may not work for SUSE os ?  take care of that

Reply:
Checking for Distro and OS versions and making use of commands